docs/local-setup/running-a-node.md

```diff
@@ -1,68 +1,64 @@
 ---
-id: running-testnet
+id: running-a-node
 title: Running a node
 sidebar_label: Running a node
 ---
 
-## Why?
+## Running a node using `nearcore`
 
-NEAR Protocol runs on a collection of publicly maintained computers (or "nodes").
+1) If you haven't already, you will need to clone [`nearcore`](https://github.com/nearprotocol/nearcore) as well as compile and run the `neard` package using Rust. Please reference [this guide](https://docs.near.org/docs/contribution/nearcore#docsNav) for assistance.
 
-You may decide to run a node of your own for a few reasons:
+2) Download the genesis file:
+    - TestNet [`genesis.json`](https://s3-us-west-1.amazonaws.com/build.nearprotocol.com/nearcore-deploy/testnet/genesis.json)
+    - BetaNet [`genesis.json`](https://s3-us-west-1.amazonaws.com/build.nearprotocol.com/nearcore-deploy/betanet/genesis.json)
+    - MainNet does not require you to download the `genesis.json` file as it is embedded into `nearcore`
 
-- To develop and deploy contracts on a node connected to `MainNet`, `TestNet` or `BetaNet` (†)
-- To develop and deploy contracts on a local (independent and isolated) node (sometimes called "LocalNet"). (††)
-- To join a network as a validator running a "validator node" (see [staking](/docs/validator/staking))
+3) Download the config file:
+    - TestNet [`config.json`](https://s3-us-west-1.amazonaws.com/build.nearprotocol.com/nearcore-deploy/testnet/config.json)
+    - BetaNet [`config.json`](https://s3-us-west-1.amazonaws.com/build.nearprotocol.com/nearcore-deploy/betanet/config.json)
+    - MainNet [`config.json`](https://s3-us-west-1.amazonaws.com/build.nearprotocol.com/nearcore-deploy/mainnet/config.json)
 
-_( † ) `TestNet` is intended to operate as closely (similarly) to `MainNet`  as possible with only stable releases while `BetaNet` follows a weekly release cycle._
+4) Run the following command replacing the paths & networkId:
+    ```bash
+    neard --home=<absolute_path_to_the_desired_location> init --chain-id=<networkId> --genesis=<absolute_path_to_the_downloaded_genesis_file>
```

Comment:
`--genesis` argument only applies to betanet and testnet. For mainnet, genesis is built into the binary

Reply:
This PR was to add running a node without using `nearup` to this document. Existing instructions on running `nearup` were not checked for accuracy. Thank you for the update.